### ISSUE TITLE
Formatter Updates Deprecated Specialization Declaration Parameters

### ISSUE THREAD

**Issue by ScottCarda-MS** [microsoft/qsharp-compiler#1202]
The formatter should have a rule for updating the specialization declaration syntax, replacing the deprecated syntax for parameters with the supported syntax.
- `body ()` should be replaced with `body (...)`.
- `adjoint ()` should be replaced with `adjoint (...)`.
- `controlled (cs)` should be replaced with `controlled (cs, ...)`
- `adjoint controlled (cs)` should be replaced with `adjoint controlled (cs, ...)`.

For example:
```qsharp
operation Foo () : Unit is Ctl + Adj {
    body () {
        // ...
    }
    adjoint () {
        // ...
    }
    controlled (q) {
        // ...
    }
    controlled adjoint (q) {
        // ...
    }
}
```
should be updated to
```qsharp
operation Foo () : Unit is Ctl + Adj {
    body (...) {
        // ...
    }
    adjoint (...) {
        // ...
    }
    controlled (q, ...) {
        // ...
    }
    controlled adjoint (q, ...) {
        // ...
    }
}
```

**Comment by yjt98765** [comment]
Then it should be easy to implement. I will create a pull request soon...

**Comment by ScottCarda-MS** [comment]
Resolved with #1226.
